javascript-exercises-2/exercise-1/exercise-1.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="./exercise-1.js" type="text/javascript"></script>
+  <title>JS DOM paragraph style</title>
+</head>
+<body>
+    <p id="text">JavaScript Exercises - w3resource</p>
+    <div>
+      <button id="jsstyle" onclick="changeText()">Style</button>
```

Comment:
You can check the format rule for Id at [link](https://google.github.io/styleguide/htmlcssguide.html#id_Attributes)